test/.eslintrc.json

```diff
@@ -1,5 +1,9 @@
 {
   "env": {
+    "es6": true,
+    "node": true,
+    "browser": true,
```

Comment:
The `node` and `browser` options seem mutually exclusive to me. Given that the tests (currently) run in Node, I think we should omit `browser`, but let me know if there's a reason to have both that I'm missing.